package.json

```diff
@@ -1,11 +1,31 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap",
-    "postinstall": "pip install -e ."
+    "bootstrap": "yarn && yarn dependencies && yarn nbstripout:install && yarn development",
+    "development": "pip install -r requirements-development.txt",
+    "nbstripout:install": "conda install -y nbstripout && nbstripout --install",
+    "dependencies": "cross-os dependencies",
+    "dependencies:darwin:win32": "conda install --file requirements-dependencies-only.txt",
+    "dependencies:linux": "export INSTALL_ON_LINUX=1 && pip install -r requirements-dependencies-only.txt",
+    "conda:create": "conda create -y --name pmp python=3.7 && yarn conda:activate && yarn bootstrap",
+    "conda:delete": "conda env remove -y --name pmp",
+    "conda:activate": "conda activate pmp"
   },
-  "devDependencies": {
-    "lerna": "^3.13.3"
+  "cross-os": {
+    "dependencies": {
+      "win32": "yarn dependencies:darwin:win32",
+      "darwin": "yarn dependencies:darwin:win32",
+      "linux": "yarn dependencies:linux"
+    }
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "nbstripout --is-installed"
```

Comment:
So once a user runs `yarn bootstrap` they will have a pre-commit hook installed which forcibly checks if `nbstripout` is installed appropriately. ... potentially that means we can remove bits from the contributor docs. What do you think?

Reply:
![image](https://user-images.githubusercontent.com/6559099/56563147-4e318200-65ee-11e9-9fc0-efdf2712f2ce.png)

Reply:
Cool! Yeah, I we could omit *nbstripout*, I think.

Once we're comfortable with the commands, I'd almost introduce the batch file and suggest running that, but retain the existing recommendations (adapted to whatever changes we make) for understanding?

Reply:
> but retain the existing recommendations (adapted to whatever changes we make) for understanding?

Potentially retain them in a separate document that is designed to describe "what magic just happened" when they run `yarn conda:create`. We can then link to that document right before asking them to ran `yarn conda:create` so those who want to do it manually, or learn what is about to happen to their machine can do so...